.appveyor.yml

```diff
@@ -10,7 +11,7 @@ environment:
 
 install:
     - SET PATH=C:\\Python%PYTHON%;c:\\Python%PYTHON%\\scripts;%PATH%
-    - python -m pip install -U pip wheel setuptools
+    - pip install -U pip wheel setuptools
```

Comment:
`python -m pip` is required on Windows if you want to update pip itself. Otherwise, `pip.exe` would be write-protected because it's in use.